src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  let partner = '';
+
+  if (person.isMarried) {
+    partner = person.sex === 'm'
+      ? `${person.partnerName} is my wife`
+      : `${person.partnerName} is my husband`;
+  } else {
+    partner = 'I am not married';
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {person.name}
```

Comment:
Use string template here, because there is no spaces and this is why your tests are failing